src/components/TaskCard/index.jsx

```diff
@@ -1,13 +1,21 @@
 import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import { deleteTask } from "../../store/actions";
+import { DELETE } from "../../utils/constants/icons";
 
 import "./index.scss"
 
 const TaskCard = ({ task }) => {
+  const dispatch = useDispatch();
+
+  const handleDelete = () => {
+    dispatch(deleteTask(task.id));
+  };
   return (
     <div className="task-card">
       <p className="task-card__title">{task.title}</p>
       <p>Created At: {task.createdDate}</p>
-      <button>Delete</button>
+      <img src={DELETE} onClick={handleDelete} />
```

Comment:
Shouldn't this be button instead of img tag?

Reply:
Used button instead of img tag. Thanks.